lib/constructorio.js

```diff
@@ -384,6 +385,121 @@ ConstructorIO.prototype = {
     }
   },
 
+  /**
+   * @description Adds a one way synonym.
+   */
+  addOneWaySynonym(params, callback) {
+    const json = clonedeep(params);
+    const { phrase } = json;
+    delete json.phrase;
+
+    request({
+      method: 'POST',
+      url: this.makeUrl(`one_way_synonyms/${phrase}`, { basePath: 'v2' }),
+      auth: this.makeAuthToken(),
+      json,
+    }, (error, response) => {
+      handleServerResponse(error, response, callback);
+    });
+  },
+
+  /**
+   * @description Modifies a one way synonym for supplied parent phrase.
+   */
+  modifyOneWaySynonym(params, callback) {
+    const json = clonedeep(params);
+    const { phrase } = json;
+    delete json.phrase;
+
+    request({
+      method: 'PUT',
+      url: this.makeUrl(`one_way_synonyms/${phrase}`, { basePath: 'v2' }),
+      auth: this.makeAuthToken(),
+      json,
+    }, (error, response) => {
+      handleServerResponse(error, response, callback);
+    });
+  },
+
+  /**
+   * @description Retrieves all one way synonyms.
+   */
+  getOneWaySynonyms(params, callback) {
+    const { num_results_per_page, phrase, page } = params;
+    const qsParams = {};
+
+    if (num_results_per_page) {
+      qsParams.num_results_per_page = num_results_per_page;
+    }
+
+    if (phrase) {
+      qsParams.phrase = phrase;
+    }
+
+    if (page) {
+      qsParams.page = page;
+    }
+
+    request({
+      method: 'GET',
+      url: this.makeUrl('one_way_synonyms', { basePath: 'v2' }),
+      auth: this.makeAuthToken(),
+      qs: qsParams,
+    }, (error, response) => {
+      handleServerResponse(error, response, callback);
+    });
+  },
+
+  /**
+   * @description Retrieves a one way synonym for supplied parent phrase.
+   */
+  getOneWaySynonym(params, callback) {
+    const json = clonedeep(params);
+    const { phrase } = json;
+    delete json.group_id;
+
+    request({
+      method: 'GET',
+      url: this.makeUrl(`one_way_synonyms/${phrase}`, { basePath: 'v2' }),
+      auth: this.makeAuthToken(),
+      json,
+    }, (error, response) => {
+      handleServerResponse(error, response, callback);
+    });
+  },
+
+  /**
+   * @description Remove all one way synonyms.
+   */
+  removeOneWaySynonyms(params, callback) {
+    request({
+      method: 'DELETE',
+      url: this.makeUrl('one_way_synonyms', { basePath: 'v2' }),
+      auth: this.makeAuthToken(),
+    }, (error, response) => {
+      handleServerResponse(error, response, callback);
+    });
+  },
+
+  /**
+   * @description Remove a one way synonym for supplied parent phrase.
```

Comment:
Super small thing but none of the other comments end in a period (.)

Reply:
Updated!